internal/pkg/cache/registry.go

```diff
@@ -48,6 +55,22 @@ func (r *Registry) get(opts *GetOpts, cache *Cache) (err error) {
 			continue
 		}
 
+		// Read the top-level metadata file but don't process it like a pack
+		if packEntry.Name() == "metadata.json" {
+			f, err2 := os.ReadFile(path.Join(opts.RegistryPath(), packEntry.Name()))
```

Comment:
Curious why we are using err2 here and not err?

Reply:
if it's an `err`, linter complains that it shadows previous err declaration.